html/elements/button.json

```
@@ -264,10 +264,10 @@
                 "version_added": null
               },
               "safari": {
-                "version_added": null
+                "version_added": true
```

Looking at [the history of the feature](https://bugs.webkit.org/show_bug.cgi?id=49240), it looks like this was implemented quite some time ago. By the date of the linked bug, I'd estimate Safari 5.1. This also aligns with the existing Chrome data here, which comes from before the Blink fork from WebKit.

```suggestion
                "version_added": "5.1"
```